Cargo.toml

```diff
@@ -74,12 +78,30 @@ pretty-bytes = "0.2.2"
 
 [features]
 blt-backend = ["bear-lib-terminal"]
-default = ["ncurses-backend"]
+default = ["crossterm"]
 markdown = ["pulldown-cmark"]
 ncurses-backend = ["ncurses", "maplit", "term_size"]
 pancurses-backend = ["pancurses", "maplit", "term_size"]
 termion-backend = ["termion"]
+crossterm-backend = ["crossterm"]
 unstable_scroll = []
 
 [lib]
 name = "cursive"
+
+[[example]]
+name="color"
+path="examples/colors.rs"
```

Comment:
You should already be able to do `cargo run --example color` without this in the manifest

Reply:
Did not no that, I will try that out.